nbm-maven-plugin/src/it/full/pom.xml

```diff
@@ -63,6 +62,14 @@ under the License.
                         </archive>
                     </configuration>
                 </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-surefire-plugin</artifactId>
+                    <version>3.2.1</version>
+                    <configuration>
+                        <argLine>-Djava.security.manager=allow --add-opens=java.base/java.net=ALL-UNNAMED</argLine>
```

Comment:
This should go into a profile that is activated like:
```
    <activation>
      <jdk>21</jdk>
    </activation>
```

Reply:
i hoped we could turn JDK 8 off but it fails on JDK 11 too, so yeah it would have to be set dynamically

Reply:
@cstamas would you like to check if I implemented the profile correctly?

Reply:
lgtm